internal/k8s/controller.go

```diff
@@ -95,6 +97,7 @@ type LoadBalancerController struct {
 	client                        kubernetes.Interface
 	confClient                    k8s_nginx.Interface
 	dynClient                     dynamic.Interface
+	restConfig                    *rest.Config
```

Comment:
Wouldn't be possible to use the `client` that we are already passing? That's what the config is used for in the end right?

Reply:
We need it to create a cert-manager api client in the cm_controller also.